src/insns/scaddr_32bit.adoc

```diff
@@ -1,18 +1,20 @@
 <<<
 
-[#CSETADDR,reftext="CSETADDR"]
```

Comment:
There are lots of references to this anchor in the tables at the back of the spec.

Reply:
this was because I was waiting for the CSR reset PR to be merged, fixed now